docker/DOCKERFILE

```diff
@@ -0,0 +1,11 @@
+LABEL maintainer=omelisra123@gmail.com
```

Comment:
Adding Labels is always a good idea.
Maybe add other labels as well like build date etc.
https://medium.com/@chamilad/lets-make-your-docker-image-better-than-90-of-existing-ones-8b1e5de950d

Reply:
good to know it. thank you